reinforcement_learning/rl_knapsack_coach_custom/rl_knapsack_coach_customEnv.ipynb

```diff
@@ -241,11 +239,11 @@
     "                        source_dir='src',\n",
     "                        dependencies=[\"common/sagemaker_rl\"],\n",
     "                        toolkit=RLToolkit.COACH,\n",
-    "                        toolkit_version='0.11.0',\n",
+    "                        toolkit_version='1.0.0',\n",
```

Comment:
are we using coach 1.0 now?

Reply:
Since we plan to migrate these notebooks to Ray, this will be changed in a future version.